packages/dds/tree/src/simple-tree/api/schemaFactory.ts

```diff
@@ -151,6 +151,16 @@ export interface SchemaFactoryObjectOptions<TCustomMetadata = unknown>
 	readonly allowUnknownOptionalFields?: boolean;
 }
 
+/**
+ * {@link SchemaFactoryObjectOptions} with experimental (alpha) extensions.
+ *
+ * @input
+ * @alpha
+ */
+export interface SchemaFactoryObjectOptionsAlpha<TCustomMetadata = unknown>
```

Comment:
Note for reviewers: everything that was in `SchemaFactoryObjectOptions` is kept alpha via this new type except `allowUnknownOptionalFields`, which is made beta.